docs/concepts/Package-Versioning.md

```diff
@@ -128,6 +128,9 @@ Avoid specifying an upper bound to version ranges to packages you don't own unle
      Will resolve to the smallest acceptable stable version.-->
 <PackageReference Include="ExamplePackage" Version="6.1" />
 
+<!-- Accepts exactly version 6.1.0. -->
+<PackageReference Include="ExamplePackage" Version="[6.1.0]" />
```

Comment:
This is opinionated but I'd this example next to the other range notations, so maybe below the floating version example.